app/containers/Tracks/index.js

```diff
@@ -127,21 +135,23 @@ export function Track({dispatchTrackDetails,
           <CustomCard>
             <If condition={!loading} otherwise={renderSkeleton()}>
               <>
-                <If condition={!isEmpty(trackName)}>
-                  <div>
-                    <T id="search_query" values={{repoName: trackName}} />
-                  </div>
-                </If>
+               
                 <If condition={totalCount !== 0}>
                   <div>
-                    <T id="matching_repos" values={{ totalCount }} />
+                    <T id="records_found" values={{ totalCount }} />
                   </div>
                 </If>
-                <For
-                  of={items}
-                  ParentComponent={Container}
-                  renderItem={(item, index) => <TrackCard key={index} {...item} />}
-                />
+                <div style={{display: 'grid', gridTemplateColumns: 'repeat(auto-fit, minmax(300px, 1fr))', gridGap: '1rem'}}>
```

Comment:
use styled components. Avoid defining styles in the component itself. Makes your code unreadable